GiEnJul/ClientApp/src/components/institution/FormPerson.tsx

```diff
@@ -1,4 +1,6 @@
-import { capitalize, Checkbox, FormControlLabel, Grid } from '@material-ui/core';
+import { IconButton } from '@material-ui/core';
+import { capitalize, Checkbox, FormControlLabel, Grid, SvgIcon } from '@material-ui/core';
```

Comment:
These are imported from the same library and you dont need to have them on two lines
should look like this -> import { capitalize, Checkbox, FormControlLabel, **IconButton**, Grid, SvgIcon } from '@material-ui/core';